src/js/datetimepicker.js

```diff
@@ -132,6 +195,9 @@
         replace: true,
         link: function link(scope, element, attrs, ngModelController) {
 
+          scope.showCurrent = attrs.showCurrent ? scope.$parent.$eval(attrs.showCurrent)
+                                                : ['year','month','day','hour','minute'];
```

Comment:
make this a boolean value - hightlightNow = true | false
Not a separate attribute but rather a part of the exiting config
default to false